Add list structure

edwardoluyara opened the pull request:
- Set up webpack for To Do List
- Add basic structure to To Do List
- Add DOM dynamically

edwardoluyara committed:
Add list structure

edwardoluyara committed:
Update README.md

edwardoluyara committed:
Fix deploy Issue

edwardoluyara committed:
Update index.js

edwardoluyara committed:
set up git workflow

edwardoluyara committed:
Add README.md file

edwardoluyara committed:
Made changes to styles.css file

edwardoluyara committed:
Update .stylelintrc.json

edwardoluyara committed:
Delete .stylelintrc.json

edwardoluyara committed:
Fix linter errors

DeliceLydia reviewed (changes requested):
## Hi @oluyaratosin123, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
 
- [ ] Please write a PR its not professional to submit a project with an empty PR
- [ ] Write README and don't forget to add how to get the app locally up and running


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

edwardoluyara committed:
set up list structure for To Do List

# Built With
  1. HTML
  2. CSS
  3. Linters (Webhint, Stylelint, Lighthouse).
  4. Webpack Module Bundler.

edwardoluyara committed:
Fix linter errors

carlos-ssh reviewed (approved):
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @oluyaratosin123 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



![Approved](https://user-images.githubusercontent.com/22258274/138466108-6b14c541-6a88-44c5-ba76-c64992ac1835.gif)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
